map/src/test/java/openlr/map/teststubs/LineStub.java

```diff
@@ -150,6 +150,14 @@ public FunctionalRoadClass getFRC() {
 
     public GeoCoordinates getGeoCoordinateAlongLine(int distance) throws MapStubException {
 
+        if(distance ==0){
```

Comment:
Could you run the test classes also through the code formatter?

Reply:
482ec5f58091993383b0ec513ceafb5223fa03d7